html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -459,8 +459,11 @@ sub authenticationLogin : Private {
         my $mac       = $portalSession->clientMac;
         my $node_info = node_view($mac);
         my $username = $node_info->{'last_dot1x_username'};
```

Comment:
In the case of status/login and if packetfence is not able to have the mac address based on the ip address then the username will be empty.